### PR TITLE
Added Database::version

### PR DESCRIPTION
**Pull request by sergeyklay**
See please http://dev.kohanaframework.org/issues/4749

### CODE REVIEW

**Comment by acoulton** [comment]
@sergeyklay thanks for this proposal and sorry it's been left open so long. Per Ivan's comment on [#R4749](http://dev.kohanaframework.org/issues/4749) could you clarify the usecase for this?

Also is there a way you could add some unit tests for the new methods?

**Comment by sergeyklay** [comment]
@acoulton I found it convenient to have a version number - for example, to create flexible installers

**Comment by kemo** [comment]
I'd drop the boolean argument

**Comment by sergeyklay** [comment]
Why?

File: classes/Kohana/Database/MySQL.php [hunk 1]
```diff
@@ -440,4 +440,23 @@ public function escape($value)
...
+	 *
+	 * @param   boolean  $full  Show full version? [Optional]
+	 * @return  string
+	 */
+	public function version($full = FALSE)
+	{
+		// Make sure the database is connected
+		$this->_connection or $this->connect();
+
+		$result = mysql_query('SHOW VARIABLES WHERE variable_name = "version"', $this->_connection);
```
Comment (cbandy) [inline classes/Kohana/Database/MySQL.php:456]: Can also use `SELECT VERSION();`.

**Comment by sergeyklay** [comment]
Unfortunately I do not have time to do this

**Pull request closed without merging**